test/atm/test_store_gsi_satbias.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+import os
+from r2d2 import store
+from solo.configuration import Configuration
+from solo.date import date_sequence
+import yaml
+import ufsda.r2d2
+
+
+def store_satbias(yaml_file):
+    config = Configuration(yaml_file)
+    ufsda.r2d2.store(config)
+
+
+if __name__ == "__main__":
+    # create a dummy R2D2 config YAML
+    r2d2_config = {
+        'databases':
+            {'test': {'cache_fetch': False,
+                      'class': 'LocalDB',
+                      'root': os.path.join(os.getcwd(), 'r2d2-test')}, },
+        'fetch_order': ['test'],
+        'store_order': ['test'],
+    }
```

Comment:
couldn't that be a call to ``genyaml`` and we keep an r2d2_config.yaml template somewhere?

Reply:
Sure, do you think that is what should be done in this and your existing test? Or stick with this approach for now? Does it make sense for the same r2d2 db to be used for both the atm and soca tests?

Reply:
I think so, we have a few r2d2_config dictionaries defined here and there, it might be a good idea to consolidate how we deal with the config. I would say create an issue somewhere so we can remember to deal with it later, or change it now? Up to you. I won't block this pr for this issue.
As for the r2d2 db, yes, I do think we should have the same one.